mint-choc-chip-skyblade/sshd-rando
# Random Starting Statues

## What does this PR do?
Adds in the recently made SDR setting of Random Starting Statues. 

## How do you test this changes?
I manually tested each statue to make sure the proper flags were set. I also tested checking the beginning statue in each area manually and the statue properly appeared on the statue selection screen afterwards.
